L1Trigger/L1TTrackMatch/plugins/L1TrackVertexAssociationProducer.cc

```diff
@@ -148,6 +148,14 @@ class L1TrackVertexAssociationProducer : public edm::global::EDProducer<> {
     std::vector<double> deltaZMaxEtaBounds_;
     std::vector<double> deltaZMax_;
   };
+  struct TTTrackLinkLimitSelector {
```

Comment:
any reason why needs to be a `struct` and it cannot be simply a free function ?

Reply:
Not anymore! Used to be stateful in a failed implementation (i.e. did not match F/W)

Reply:
Replaced with a free function now